flake.nix

```diff
@@ -25,7 +31,8 @@
       in rec {
         packages.helix = naerskLib.buildPackage {
           pname = "helix";
-          root = ./.;
+          root = inputs.helix;
```

Comment:
There really must be some better way? :/

Reply:
Needs fixing in Nix :/ https://github.com/NixOS/nix/issues/4423

Reply:
I did try a `symlinkJoin` and it mostly worked actually. But not if you add new submodules, and it seemed kinda hacky so I left it out.

Reply:
Wouldn't symlinkJoin allow avoiding `inputs` altogether and would build package from cloned version?

Reply:
How do you mean?

Reply:
Ahhh, I see what you did there :D I'd say that it should be left as it is